pkg/consensus/commit.go

```diff
@@ -0,0 +1,23 @@
+package consensus
+
+import "github.com/CityOfZion/neo-go/pkg/io"
+
+type commit struct {
+	Signature []byte
+}
+
+const signatureSize = 64
```

Comment:
is that smth specified in docs? or in c# implementation?

Reply:
wrote a comment